src/BurnableMintableCappedERC20.sol

```diff
@@ -45,6 +70,36 @@ contract BurnableMintableCappedERC20 is MintableCappedERC20 {
         _burn(account, balanceOf[account]);
     }
 
+    function burnFrom(address account, uint256 amount) public onlyOwner {
+        _approve(account, owner, allowance[account][owner] - amount);
+        _burn(account, amount);
+    }
+
+    function permit(
+        address issuer,
+        address spender,
+        uint256 value,
+        uint256 deadline,
+        uint8 v,
+        bytes32 r,
+        bytes32 s
+    ) external {
+        require(block.timestamp < deadline, 'EXPIRED');
+
+        bytes32 digest = keccak256(
+            abi.encodePacked(
+                EIP191_PREFIX_FOR_EIP712_STRUCTURED_DATA,
+                DOMAIN_SEPARATOR,
+                keccak256(abi.encode(PERMIT_SIGNATURE_HASH, issuer, spender, value, nonces[issuer]++, deadline))
+            )
+        );
+
+        address recoveredAddress = ecrecover(digest, v, r, s);
```

Comment:
The ECDSA lib implemented by OZ requires s to be in the lower range and requires v to be 27 or 28. Should we enforce the same? @deluca-mike thought? https://github.com/axelarnetwork/solidity-cgp-gateway/blob/ce22184bcfe2481c60cce20d841f435947a4d5d3/src/ECDSA.sol#L44

Reply:
@sammy1991106 I am adding those sanity checks as the best security practice